test/tests/relationship_managers/two_rm/tests

```diff
@@ -14,4 +14,22 @@
     requirement = "MOOSE shall allow multiple active RelationshipManagers to manage ghosting for the same object"
     issues = "#10455 #11981"
   [../]
+
+  [./evaluable_neighbors_distributed]
+    type = 'Exodiff'
+    input = 'two_rm.i'
+    exodiff = 'two_rm_dist_out.e'
+    cli_args = 'Mesh/parallel_type=distributed Outputs/file_base=two_rm_dist_out'
```

Comment:
So we can do this for geometric ghosting/distributed mesh now? Good :smile:

Reply:
We sure can - that's where this actually matters. I do still plan to look at optimizing the addition of functors in the first place. I don't think it'll make a difference in terms of optimization, but I like leaving them off conceptually.